### DISCUSSION TITLE
scratch file path | journalctl -u libvirtd.service | Path '[...]/backup.XL0A3.vda' is not accessible: No such file or directory

### DISCUSSION THREAD

**Discussion by slu-010101** [abbbi/virtnbdbackup#204] [General]
Hi,

I'm not sure this has something to do with virtnbdbackup but I see every backup the following error in libvirtd.service:
```
Aug 18 09:28:48 kvmd12 libvirtd[1537]: Path '/mnt/storage/virtnbdtmp/backup.XL0A3.vda' is not accessible: No such file or directory
Aug 18 09:28:48 kvmd12 libvirtd[1537]: Unable to tear down cgroup access on /mnt/storage/virtnbdtmp/backup.XL0A3.vda
Aug 18 09:28:48 kvmd12 libvirtd[1537]: internal error: child reported (status=125): unable to stat: /mnt/storage/virtnbdtmp/backup.XL0A3.vda: No such file or directory
Aug 18 09:28:48 kvmd12 libvirtd[1537]: unable to stat: /mnt/storage/virtnbdtmp/backup.XL0A3.vda: No such file or directory
Aug 18 09:28:48 kvmd12 libvirtd[1537]: internal error: child reported (status=125): unable to stat: /mnt/storage/virtnbdtmp/backup.XL0A3.vda: No such file or directory
Aug 18 09:28:48 kvmd12 libvirtd[1537]: unable to stat: /mnt/storage/virtnbdtmp/backup.XL0A3.vda: No such file or directory
Aug 18 09:28:48 kvmd12 libvirtd[1537]: Unable to restore security label on /mnt/storage/virtnbdtmp/backup.XL0A3.vda
```
I have no vm called "backup" so this must be some other temporary file?
Any idea whats wrong here?
virtnbdbackup doesn't show any error and the backup is recoverable.

Debian 12.6
ii  libvirt-daemon                       9.0.0-4                                amd64        Virtualization daemon
ii  virtnbdbackup                        2.9-1                                  all          Backup utility for libvirt

**Comment by abbbi** [comment]
Known non-issue, the scratch file removal is handled by libvirt, i dont know why this warning Happens but it’s not an issue

it seems it wants to remove cgroup informations but the scratch file is already gone, there was another discussion/issue already about this afakr

**Reply by slu-010101** [reply]
Thank you for reply, happy my backup is fine.
